### PR TITLE
added ultimate topics list for competitive programming

### PR DESCRIPTION
**Pull request by HridoyHazard**
## What does this PR do?
Add Resource
Add resource(s) | Remove resource(s) | Add info | Improve repo

## For resources
https://blog.shahjalalshohag.com/topic-list/
### Description
The Ultimate Topic List(with Tutorials, Problems, and Templates). everything at once. it will surely help the programmer who are aimless in competitive programming
## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/problem-sets-competitive-programming.md [hunk 1]
```diff
@@ -120,5 +120,6 @@
 * [Sphere Online Judge](http://www.spoj.com/problems/classical)
 * [TalentBuddy](http://www.talentbuddy.co/blog/)
 * [Timus Online Judge](http://acm.timus.ru)
+* [The Ultimate Topic List(with Tutorials, Problems, and Templates)](https://blog.shahjalalshohag.com/topic-list/)
```
Comment (davorpa) [inline more/problem-sets-competitive-programming.md:123]:
```suggestion
* [The Ultimate Topic List(with Tutorials, Problems, and Templates)](https://blog.shahjalalshohag.com/topic-list/) - Shahjalal Shohag
```

File: more/problem-sets-competitive-programming.md [hunk 2]
```diff
@@ -120,5 +120,6 @@
 * [Sphere Online Judge](http://www.spoj.com/problems/classical)
 * [TalentBuddy](http://www.talentbuddy.co/blog/)
 * [Timus Online Judge](http://acm.timus.ru)
+* [The Ultimate Topic List(with Tutorials, Problems, and Templates)](https://blog.shahjalalshohag.com/topic-list/) - Shahjalal Shohag
```
Comment (davorpa) [inline more/problem-sets-competitive-programming.md:123]: Alphabetize

```suggestion
* [The Ultimate Topic List(with Tutorials, Problems, and Templates)](https://blog.shahjalalshohag.com/topic-list/) - Shahjalal Shohag
* [Timus Online Judge](http://acm.timus.ru)
```

File: more/problem-sets-competitive-programming.md (hunk 1 above)
Comment (HridoyHazard) [reply more/problem-sets-competitive-programming.md]: what's wrong with the commit?? can you plz tell me in details. from my end everything seems ok

**Review by davorpa — APPROVED** [review]

File: more/problem-sets-competitive-programming.md (hunk 2 above)
Comment (HridoyHazard) [reply more/problem-sets-competitive-programming.md]: thanks a ton @davorpa  really appreciate your help on this commit.

**Comment by HridoyHazard** [comment]
@davorpa can you plz add the hacktoberfest label in this pr ? it will be helpful for me

**Comment by davorpa** [comment]
> @davorpa can you plz add the hacktoberfest label in this pr ? it will be helpful for me

`hacktoberfest-accepted` is not needed since repo is tagged as `hacktoberfest`. It will count anyway :wink:

**Pull request merged**